### ISSUE TITLE
In-project comments not saving

### ISSUE THREAD

**Issue by DrDingleberry** [LLK/scratch-gui#5638]
### Expected Behavior

The in project comments are to attach to a block and display the information that the user typed in.

### Actual Behavior

The comments all disappear upon opening the project after leaving.

### Steps to Reproduce

Create a project, then create a comment be right-clicking on a block, and typing into the new text bubble. Save and then leave the page, then return.

### Operating System and Browser

Windows 10 Home x64 version 1909, Google Chrome 81.0.4044.92

**Comment by towerofnix** [comment]
To help reproduce this bug more easily, are you noticing this when the comment is attached to a particular description of blocks? (Variable reporters, stack blocks, one specific block, etc.) Or does it happen when attached to any block?

**Comment by BryceLTaylor** [comment]
Just to double check, have you saved the project after adding the comment?  So far I've been unable to reproduce the issue.

**Comment by DrDingleberry** [comment]
> 
> 
> To help reproduce this bug more easily, are you noticing this when the comment is attached to a particular description of blocks? (Variable reporters, stack blocks, one specific block, etc.) Or does it happen when attached to any block?

Yeah it happens particularly after it is attached to a block and resized. I also noticed recently that switching to the costume tab causes the comments to disappear.

**Comment by DrDingleberry** [comment]
> 
> 
> Just to double check, have you saved the project after adding the comment? So far I've been unable to reproduce the issue.

In order to get the comments to save, I would write one, save, leave the project, then come back. I've edited my post to contain extra details and things I've noticed.

**Comment by DrDingleberry** [comment]
> 
> 
> To help reproduce this bug more easily, are you noticing this when the comment is attached to a particular description of blocks? (Variable reporters, stack blocks, one specific block, etc.) Or does it happen when attached to any block?

Seems to be any block.

**Comment by BryceLTaylor** [comment]
Hmm... I'm still unable to reproduce the issue.  Can you write down every step you take.  Including when you change tabs, which blocks you drag out, what menus you open, etc. Start from a brand new project.

**Comment by qucchia** [comment]
I tried to reproduce this, what happened to me was the comments were not visible but appeared after a couple seconds.
![ezgif-5-eb2b97104273](https://user-images.githubusercontent.com/45072410/79612715-7b2f4c80-80fd-11ea-8d68-f5633ade3636.gif)

I'm on Google Chrome OS Version 81.0.4044.94 (Official Build) beta (64-bit)
